### DISCUSSION TITLE
Pass variable to View::make()

### DISCUSSION THREAD

**Discussion by sh4msi** [filamentphp/filament#4421] [Help] [answered]
hi,

How can I pass a variable to View::make() in Resource?

**Comment by danharrin** [comment, marked as answer]
What do you want to pass?

**Reply by sh4msi** [reply]
Data received from the database and changes made in it
(data of a model)

**Reply by danharrin** [reply]
Inside the view, the saved database model is `$getRecord()` and the current unsaved form data can be retrieved with `$evaluate(fn ($get) => $get('field_name_here'))`

**Reply by sh4msi** [reply]
If I want to access a function, how should I act?
(Error returned: $evaluate(fn ($get) => myFunc()) )

**Reply by danharrin** [reply]
What function? Where is it defined? You don't need to `$evaluate()` that

**Reply by sh4msi** [reply]
In the resource

**Reply by danharrin** [reply]
resource methods arent accessible like that... it would need to be on the Livewire component (Create page, Edit page, View page) class, which is accessible with `$this->myFunc()` from the view
